torch/_inductor/lowering.py

```diff
@@ -2621,6 +2621,10 @@ def is_aligned(x):
 make_fallback(aten._pdist_forward)  # Has decomp. Needs benchmarks
 make_fallback(aten.soft_margin_loss_backward, warn=False)  # py_impl?
 make_fallback(aten._fused_rms_norm, warn=False)  # (MPS-only and faster than decomp)
+if torch.xpu.is_available():
+    make_fallback(
+        aten.embedding_dense_backward, warn=False
+    )  # (XPU-only and faster than decomp)
```

Comment:
It seems like a patch. I'm considering that we may need to provide a device-specific fallback design. Meanwhile,  @jianyizh , pls. add another section dedicated to XPU besides `1) Easy 2) Medium ...` 
https://github.com/pytorch/pytorch/pull/151637/files#diff-a1b077971cddfabfa0071c5162265066e867bc07721816d95b9cbe58431c38e3R2618